lib/pinboard_tabs_downloader.rb

```diff
@@ -0,0 +1,89 @@
+require 'fileutils'
+require 'pathname'
+require_relative 'downloader/config'
+require_relative 'downloader/scraper'
+
+class PinboardTabsDownloader
+
+  attr_reader :scraper, :all_episodes
+  def initialize
+    @lib_root     = File.expand_path(File.join(File.dirname(__FILE__),'..'))
```

Comment:
Unnecessary spacing detected.<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.